src/control/cmd/daos/container.go

```diff
@@ -301,29 +301,31 @@ func (cmd *containerCreateCmd) Execute(_ []string) (err error) {
 	if err != nil {
 		return err
 	}
+
+	var co_id string
```

Comment:
nit: snake_case variable names will cause linter warnings for Go developers. Can be fixed later though.